code/modules/integrated_electronics/core/assemblies/clothing.dm

```diff
@@ -0,0 +1,192 @@
+
+// The base subtype for assemblies that can be worn. Certain pieces will have more or less capabilities
+// E.g. Glasses have less room than something worn over the chest.
+// Note that the electronic assembly is INSIDE the object that actually gets worn, in a similar way to implants.
+
+/obj/item/device/electronic_assembly/clothing
+	name = "electronic clothing"
+	icon_state = "circuitry" // Needs to match the clothing's base icon_state.
+	desc = "It's a case, for building machines attached to clothing."
+	w_class = ITEMSIZE_SMALL
+	max_components = IC_COMPONENTS_BASE
+	max_complexity = IC_COMPLEXITY_BASE
+	var/obj/item/clothing/clothing = null
+
+/obj/item/device/electronic_assembly/clothing/ui_host()
```

Comment:
UI functions but no included UI files? A dubious proposition, to be certain.

Reply:
It uses the normal circuit UI files, I believe.

Reply:
I thiiiiink it's fine, because the circuit UI does open properly with circuit clothes.